### ISSUE TITLE
Issue: using Fine-tuning model with Langchain

### ISSUE THREAD

**Issue by daniyal-arif-tps** [langchain-ai/langchain#8057]
### Issue you'd like to raise.

Hi, I am using a fine tuned model that generates user prompts to SQL queries, instead of the default model provided by langchain. The reason for doing this is because langchain does not know about all the data from the database unless you provide context to it but there is a lot of data hence why it can create incorrect SQL queries also it is unable to form complex queries for similar reasons even after you give it context. 

So my question is based upon the output I am getting below. Is there a way to keep the initial question I asked the same throughout like in the action input, instead of "SOME_BRANCH_NAME" I want the entire sentence to go through to the SQLDatabaseChain like the user initially asked which is "what is the summary of last 3 issues reported by SOME_BRANCH_NAME". Basically since the Action Input is different from what the user asked, it is generating the wrong SQL query since what it should be doing is this, "SELECT summary FROM sla_tat_summary WHERE organization like '%SOME_BRANCH_NAME%' ORDER BY ReportedDate DESC LIMIT 3;" instead of what is shown below. I could just use the SQLDatabaseChain on its own which does get the exact query I want since I was able to make sure only the prompt the user gave went through, but the agent is needed since I am using it for things other than SQL generation. 

user prompt:  what is the summary of last 3 issues reported by SOME_BRANCH_NAME 
> Entering new AgentExecutor chain...
 I need to find out what the last 3 issues reported by SOME_BRANCH_NAME were.
Action: TPS Issue Tracker Database
Action Input: SOME_BRANCH_NAME

> Entering new SQLDatabaseChain chain...
SOME_BRANCH_NAME:
SELECT organization, COUNT(*) FROM sla_tat_summary WHERE severity = 'Level 2 - Critical' GROUP BY organization ORDER BY COUNT(*) DESC LIMIT 1;

### Suggestion:

In summary send user prompt as it is to Action Input.
